src/index.js

```diff
@@ -1,21 +1,17 @@
 "use strict";
 
-let WritableStream = require("./writable_stream");
+const WritableStream = require("./writable_stream");
```

Comment:
It's cure how y'all think this is a democracy, or that rational arguments might sway me.

Seriously though, let's just adhere to the surrounding code here and discuss stylistic concerns elsewhere (or, you know, not at all).